src/messages.js

```diff
@@ -12,6 +12,7 @@ export default {
   'column.collectionobject.narrow.objectNumber': 'Acc. number',
 
   'field.collectionobjects_common.objectNumber.name': 'Accession number',
+  'field.collectionobjects_common.objectCountGroup.name': 'Number of objects',
```

Comment:
For the search page, the `field.collectionobjects_common.objectCount.fullName` and `field.collectionobjects_common.objectCount.name` messages need to be set to 'Number of objects'. You don't need to change `field.collectionobjects_common.objectCountGroup.name`, because of the trick...